runtime/src/main/java/com/tns/AssetExtractor.java

```diff
@@ -32,4 +41,33 @@ public void extractAssets(Context context, String inputPath, String outputPath,
             }
         }
     }
+
+    /**
+     * Delete a file or a directory and its children.
+     * @param file The directory to delete.
+     * @throws IOException Exception when problem occurs during deleting the directory.
+     */
+    private static void delete(File file) throws IOException {
+        File[] files = file.listFiles();
+        if (files == null) {
+            Log.d(LogTag, "Can't remove previously installed assets in " + file.getAbsolutePath());
+            return;
+        }
+
+        for (File childFile : files) {
```

Comment:
Not in Java. Java is special! There is a more concise API to work with files, but it isn't available on devices up to Level 23